megazords/ios/MozillaAppServicesTests/NimbusTests.swift

```diff
@@ -293,8 +299,12 @@ class NimbusTests: XCTestCase {
         """)
         try nimbus.applyPendingExperimentsOnThisThread()
 
-        // Record the exposure event in Glean
-        nimbus.recordExposure(experimentId: "test-experiment")
+        // Record the valid exposure event in Glean
+        nimbus.recordExposureEvent(featureId: "test-feature")
+
+        // We will also attempt to record an invalid feature exposure to ensure it doesn't
+        // get recorded
+        nimbus.recordExposureEvent(featureId: "not-a-feature")
```

Comment:
Should this be accompanied with a negative test, called before the positive test?